german/hoelzernes-Paddel/hoelzernes-Paddel.md

```diff
@@ -0,0 +1 @@
+Das hoelzerne Paddel erweist sich als sehr marode. Damit schaffen Sie es niemals bis ans Ufer. Nichtsdestotrotz sind Sie vorangekommen und sehen nun schon das Ufer. Sie hören Musik.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.